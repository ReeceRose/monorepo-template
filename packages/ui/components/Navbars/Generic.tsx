export const GenericNavbar = (): JSX.Element => {
  return (
    <nav className="fixed top-0 z-50 flex flex-wrap items-center justify-between w-full px-2 py-3 bg-white shadow navbar-expand-lg">
      <div className="container flex flex-wrap items-center justify-between px-4 mx-auto">
        <div className="relative flex justify-between w-full lg:w-auto lg:static lg:block lg:justify-start">
          <div className="flex items-center text-2xl font-black text-gray-800 uppercase align-middle dark:text-white">
            Client - GraphQL
          </div>
        </div>
      </div>
    </nav>
  );
};
