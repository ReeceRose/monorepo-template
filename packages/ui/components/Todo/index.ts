export * from './Item';
export * from './List';
export * from './New';
