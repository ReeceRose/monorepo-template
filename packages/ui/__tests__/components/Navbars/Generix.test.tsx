import { render } from '@testing-library/react';

import { GenericNavbar } from '../../../components/Navbars';

test('GenericNavbar component matches snapshot', () => {
  const { asFragment } = render(<GenericNavbar />);
  expect(asFragment()).toMatchSnapshot();
});
