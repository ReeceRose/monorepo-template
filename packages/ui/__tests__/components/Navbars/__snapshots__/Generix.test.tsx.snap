// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`GenericNavbar component matches snapshot 1`] = `
<DocumentFragment>
  <nav
    class="fixed top-0 z-50 flex flex-wrap items-center justify-between w-full px-2 py-3 bg-white shadow navbar-expand-lg"
  >
    <div
      class="container flex flex-wrap items-center justify-between px-4 mx-auto"
    >
      <div
        class="relative flex justify-between w-full lg:w-auto lg:static lg:block lg:justify-start"
      >
        <div
          class="flex items-center text-2xl font-black text-gray-800 uppercase align-middle dark:text-white"
        >
          Client - GraphQL
        </div>
      </div>
    </div>
  </nav>
</DocumentFragment>
`;
